package.json

```diff
@@ -75,11 +75,12 @@
     "husky": "8.0.3",
     "is-ci": "3.0.1",
     "lerna": "7.4.2",
+    "lint-staged": "14",
     "netlify-cli": "16.9.3",
     "npm-run-all2": "5.0.2",
     "nx": "17.3.2",
     "postcss": "8.4.38",
-    "prettier": "2.8.8",
+    "prettier": "3.2.5",
     "pretty-quick": "3.3.1",
```

Comment:
We can now remove `pretty-quick`, right?

Reply:
Well, we can consider this. `pretty-quick` is maintained again and `v4` supporting Prettier v3 is out. But I think the `lint-staged` is the most used variant for this case, so I think it will be better to use it and prefer it over `prettier-quick`.

Reply:
Removed in https://github.com/lmc-eu/spirit-design-system/pull/1431